components/sankey/elements/legend.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Legend
+page_title: Sankey Legend
+description: Legend of the Sankey Diagram for Blazor.
+slug: sankey-legend
+tags: telerik,blazor,sankey,diagram,chart,legend
+published: True
+position: 11
+---
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+# Sankey Legend
+
+The Telerik Sankey Diagram for Blazor can show a legend, which is a visual guide with details about the nodes. This article explores how to customize the Sankey legend.
+
+## Basic Customization
+
+To customize the legend in the Sankey chart, declare a `<SankeyLegend>` tag as a direct child of `<TelerikSankey>`. The `<SankeyLegend>` tag has the following parameters:
```

Comment:
For SEO purposes, consider going with `Blazor Sankey chart` here. Note that this is the first instance of Sankey chart, everywhere else it's been Sankey diagram.